<launch>
    <arg name="config_path" default = "$(find uv_feature_tracker)/../config/void/void_config.yaml" />
    <arg name="vins_path" default = "$(find uv_feature_tracker)/../config/../" />

    <node name="feature_tracker" pkg="uv_feature_tracker" type="feature_tracker" output="screen">
        <param name="config_file" type="string" value="$(arg config_path)" />
        <param name="vins_folder" type="string" value="$(arg vins_path)" />
    </node>
    <node name="vins_estimator" pkg="uv_slam" type="vins_estimator" output="screen">
       <param name="config_file" type="string" value="$(arg config_path)" />
       <param name="vins_folder" type="string" value="$(arg vins_path)" />
       <!-- launch-prefix="gdb -ex run --><!--args" -->
    </node>

    <node name="pose_graph" pkg="pose_graph" type="pose_graph"> <!-- output="log" -->
        <param name="config_file" type="string" value="$(arg config_path)" />
        <param name="visualization_shift_x" type="int" value="0" />
        <param name="visualization_shift_y" type="int" value="0" />
        <param name="skip_cnt" type="int" value="0" />
        <param name="skip_dis" type="double" value="0" />
    </node>

</launch>
